### PR TITLE
[f] 5 - Setup tox for multiple python versions testing

### PR DESCRIPTION
**Pull request by tklarryonline**
Solved #5

### CODE REVIEW

**Comment by tklarryonline** [comment]
@anhhuy1605, @hieueastagile and @giang271291 Please take a look.

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,4 @@ var/
 
 # Testing
 .coverage
+.tox
```
Comment (hieueastagile) [inline .gitignore]: @tklarryonline Could we add it into `guide` or `tips`, should be nice for everyone :P.

**Comment by hieueastagile** [comment]
LGTM ^^.

**Comment by anhhuy1605** [comment]
LGTM

**Comment by giang271291** [comment]
Good to go

**Comment by tklarryonline** [comment]
Please click merge then. I'm writing the tutorial for applying `tox` to our `tips` :)

**Pull request merged**